css-layout-classes/index.html

```diff
@@ -0,0 +1,140 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title> CSS LAYOUT CLASSES </title>
+  <link href="styles.css" type="text/css" rel="stylesheet" />
+</head>
+
+<body>
+  <div class="container">
+    <div class="row">
+      <div class="column-full">
+        <h1> Choose your character! </h1>
+      </div>
+    </div>
+
+    <div class="super-smash">
+      <div class="row">
+        <div class="column-one-third">
+          <img src="images/link.png" alt="link">
+        </div>
+        <div class="column-one-third">
+          <article>
+            <div class="row">
+              <div class="column-full">
+              </div>
+            </div>
+            <div class="row">
+              <div class="column-one-half">
+                <h2> Super Smash Bros. Ultimate </h2>
+              </div>
```

Comment:
Currently this is inside a `column-one-third` `div`, but it should be seperate. In this example, `column` are sibling elements, and shouldn't be nested inside each other.